lib/internal/util.js

```diff
@@ -187,3 +191,40 @@ exports.convertToValidSignal = function convertToValidSignal(signal) {
 
   throw new Error('Unknown signal: ' + signal);
 };
+
+const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
+function promisify(orig) {
+  if (typeof orig !== 'function') {
+    throw new TypeError('The first argument to promisify() must be a function');
```

Comment:
I was thinking maybe we can just drop `first`. When I read the error message, my mind understood that as there are more than one argument. Not a big deal though.